ajax/libs/dna.js/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "dna.js",
+  "autoupdate": {
+   "source": "git",
+   "target": "git://github.com/dnajs/dna.js.git",
+   "fileMap": [
+    {
+      "basePath":  "",
+      "files": [
+        "dna.min.js",
+        "dna.js"
+      ]
+    }
+  ]
+}
```

Comment:
@karan-96 if you are working locally, it'll be better to squash the commits, and rebase the branch with the latest master branch on your side :)